apps/apps.humanatlas.io/src/app/pages/us1/services/predictions.service.ts

```diff
@@ -95,13 +104,21 @@ export class PredictionsService {
     );
   }
 
+  /**
+   * Loads sample JSON file
+   * @returns File promise
+   */
+  async loadSampleFile(): Promise<File> {
+    const content$ = this.http.get(SAMPLE_FILE_URL, { responseType: 'text' });
+    const content = await firstValueFrom(content$);
+    return new File([content], 'sample.json', { type: 'application/json' });
+  }
+
   /**
    * Gets supported organs
    * @returns String array observable
    */
-  loadSupportedReferenceOrgans(): Observable<string[]> {
-    return this.http
-      .get<SupportedOrgans[]>(`${this.endpoint}/supported-reference-organs`)
-      .pipe(map((organs) => organs.map((organ) => organ.id)));
+  loadSupportedReferenceOrgans(): Observable<SupportedOrgans[]> {
+    return this.http.get<SupportedOrgans[]>(`${this.endpoint}/supported-reference-organs`);
```

Comment:
We need to update the requests to use the `HraPopService` from `@hra-api/client` instead of doing the http calls ourselves